node/src/commons/arguments.js

```diff
@@ -31,5 +31,5 @@ const argumentsMap = {
 module.exports = {
     argumentsMap,
```

Comment:
I think `argumentsMap` no longer needs to be exported

Reply:
indeed e82a080